lua/mapvote/client/modules/net.lua

```diff
@@ -38,8 +38,9 @@ end )
 net.Receive( "MapVote_PlayerChangedVote", function()
     local ply = net.ReadEntity() --[[@as Player]]
     local mapIndex = net.ReadUInt( 32 )
+    local voteMult = net.ReadUInt( 32 )
```

Comment:
this can probably be less than 32 
![image](https://github.com/user-attachments/assets/6c43ce17-5951-45c6-97be-258152c3080b)
i dont think anyones gonna need a `4,294,967,295` multiplier

Reply:
could probably go as low as 8, just make sure you specify the max in the config schema